src/mpi/comm/comm_split_type.c

```diff
@@ -12,6 +12,11 @@
 #include "hwloc.h"
 #endif
 
+#ifdef HAVE_NETLOC
+#include "netloc_util.h"
+#include <math.h>
```

Comment:
What's `math.h` for in this case?

Reply:
I had added it for torus split, removed it for this patch.